onnxruntime/core/session/environment.cc

```diff
@@ -34,13 +34,14 @@
 #if defined(ENABLE_TRAINING_OPS)
 #include "orttraining/core/graph/training_op_defs.h"
 #endif
+#ifdef ENABLE_TRAINING_CORE
+#include "orttraining/core/graph/loss_function_registry.h"
```

Comment:
do you need these for ORTModule build? I thought it is only useful for training c++ api, but ENABLE_TRAINING_CORE seems to be shared by both side.

Reply:
I think they are needed (will check). However, we dont do any dedicated build for ortmodule, we simply do a full training build, so will need these.

Reply:
I believe  "orttraining/core/graph/loss_function_registry.h" is not needed for training_api (on-device training)